web/lib/icons/bold-icon.tsx

```diff
@@ -0,0 +1,20 @@
+// from Feather: https://feathericons.com/
+export default function BoldIcon(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
Why use a default export? Seems better to be consistent with the rest of the codebase.

Reply:
it's how all the other svgs work, and it matches how the svgs would be imported if we saved these as .svg rather than .tsx and used an svg loader like [svgr](https://react-svgr.com/docs/webpack/)